crates/telio-network-monitors/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+#![deny(missing_docs)]
+
+//! Network monitor module
+pub mod monitor;
+
+#[cfg(any(target_os = "macos", target_os = "ios", target_os = "tvos"))]
+/// Utility to get monitor network on apple
+pub mod apple;
+
+#[cfg(target_os = "linux")]
+/// Utility to get monitor network on linux
+pub mod linux;
+
+#[cfg(target_os = "windows")]
+/// Utility to get monitor network on windows
+pub mod windows;
+
+/// Get local interfaces
+pub mod local_interfaces;
+pub use local_interfaces::*;
```

Comment:
is this `use` needed?

Reply:
Yes as local_interfaces is exported out to be used by `local.rs`. This module is as a monitor util.

Reply:
why don't you import what you need from `local.rs`?